packages/sku/lib/configure.js

```diff
@@ -84,7 +84,9 @@ module.exports = async () => {
   await ensureGitignore({
     filepath: getPathFromCwd('.prettierignore'),
     comment: 'managed by sku',
-    patterns: lintIgnorePatterns.map(convertToForwardSlashPaths),
+    patterns: lintIgnorePatterns
+      .concat('pnpm-lock.yaml')
+      .map(convertToForwardSlashPaths),
```

Comment:
Feel like `pnpm-lock.yaml` should be considered earlier in here so we dont even lint it (rather than focusing on prettier).

Maybe shifting it up to the `lintIgnorePatterns` array?